src/auth.c

```diff
@@ -187,8 +192,9 @@ int calc_hmac(const void *data, size_t datalen,
 
 /* test HMAC
  */
-int verify_hmac(const void *data, size_t datalen,
-	hashid hid, unsigned char *hmac, char *key, int keylen)
+int
+verify_hmac(const void *data, size_t datalen, hashid hid, unsigned char *hmac,
+             char *key, int keylen)
```

Comment:
Too much space here

Reply:
Ohhhh there are three function headers for `verify_hmac()`, depending on CPP defines :/